modules/openthread/Kconfig.features

```diff
@@ -212,6 +212,10 @@ config OPENTHREAD_NEIGHBOR_DISCOVERY_AGENT
 config OPENTHREAD_NETDATA_PUBLISHER
 	bool "Thread Network Data publisher"
 
+config OPENTHREAD_OPERATIONAL_DATASET_AUTO_INIT
+	bool "Operational dataset auto init"
+	default y
```

Comment:
OpenThread default is 0, should we default to `n` here?

Reply:
Doing so causes network forming fail. So Bsim tests won't pass.